### PR TITLE
IOS-4520 update passcode options -- LTHPasscodeViewController

### PR DESCRIPTION
**Pull request by ningh1111**
Related to:
https://github.com/meganz/iOS_dev/pull/1170
https://jira.developers.mega.co.nz/browse/IOS-4520

### CODE REVIEW

**Review by lhr000lhrmega — APPROVED** [review]

File: LTHPasscodeViewController/LTHPasscodeViewController.m [hunk 1]
```diff
@@ -57,6 +57,15 @@
...
 #endif
 
+#import "MEGAChatSdk.h"
+#ifdef MNZ_SHARE_EXTENSION
+#import "MEGAShare-Swift.h"
+#elif MNZ_PICKER_EXTENSION
+#import "MEGAPicker-Swift.h"
+#else
+#import "MEGA-Swift.h"
+#endif
```
Comment (simonwng) [inline LTHPasscodeViewController/LTHPasscodeViewController.m]: Why we need to import our class into a third party library? it is we to depend on a 3rd party library, not the other way around.

**Review by lhr000lhrmega — APPROVED** [review]

File: LTHPasscodeViewController/LTHPasscodeViewController.m (hunk 1 above)
Comment (ningh1111) [reply LTHPasscodeViewController/LTHPasscodeViewController.m]: I'm using UIColor.mnz_green00A886. And it need #import "MEGA-Swift.h"
Should I make the color the other way and remove the import?

Comment (simonwng) [reply LTHPasscodeViewController/LTHPasscodeViewController.m]: You can create the color inside this library. Even hardcode it is OK.

Comment (simonwng) [reply LTHPasscodeViewController/LTHPasscodeViewController.m]: Just imagine this is an open source library, which should not aware any mega things.

Comment (ningh1111) [reply LTHPasscodeViewController/LTHPasscodeViewController.m]: Got it. Just update

**Review by lhr000lhrmega — APPROVED** [review]

**Pull request merged**